fileio/easyio.go

```diff
@@ -60,6 +60,13 @@ func EasyNextRealLine(file *EasyReader) (string, bool) {
 	return NextRealLine(file.BuffReader)
 }
 
+func EasyRemove(filename string) {
+	err := os.Remove(filename)
+	if err != nil {
```

Comment:
You don't need this "if" statement wrapped around common.ExitIfError(), inside the function it checks for nil and then dies if it is not nil (and otherwise the program carries on).

Reply:
great catch! fixed in next commit